tests/aruba/aruba.md

```diff
@@ -0,0 +1,50 @@
+# Aruba-rspec testframework for CB CLI
+The aruba test project is located in tests/aruba/
+
+## E2E tests
+
+Run without Docker:
+```
+curl -sSL https://get.rvm.io | bash -s stable --ruby
+source /Users/mhalmy/.rvm/scripts/rvm
+gem install rspec
+gem install aruba
+gem install aruba-rspec
+gem install json
+gem install rspec-json_expectations
+gem install rspec_junit_formatter
+#Edit localvars
```

Comment:
```
gem install rspec \
aruba \
aruba-rspec \
json \
rspec-json_expectations \
rspec_junit_formatter
```

Reply:
Where can I find `localvars` and which are these? User need to stand in that folder to can apply `source localvars` successfully.
Please make a link for this or list these here as well.